### PR TITLE
Full review of repository code by Vitaliy Havrona

### PR DESCRIPTION
**Pull request by Xavtso**

### CODE REVIEW

File: client/csletmelearn/src/main.jsx [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+import store from './store/reducer.js'
 import './index.css'
+import { GoogleOAuthProvider } from "@react-oauth/google"
 
 ReactDOM.createRoot(document.getElementById('root')).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <Provider store={store}>
+    <GoogleOAuthProvider clientId="520624853757-rpmqbau5gpqeg7lbdfh6t5ade49og2d8.apps.googleusercontent.com">
```
Comment (Xavtso) [inline client/csletmelearn/src/main.jsx:11]: Hardcoded client-secrets - try to use .env

File: client/csletmelearn/src/pages/DesignGenerator.jsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                  // navigate()
+              }catch(e){
+                  console.log(e)
+              }finally{
+                  setLoading(false)
+              }
+          }
+          else{
+              try{  
+                  console.log("fdd")
```
Comment (Xavtso) [inline client/csletmelearn/src/pages/DesignGenerator.jsx:64]: useless console.log()

File: client/csletmelearn/src/main.jsx (hunk 1 above)
Comment (fdsssawe) [reply client/csletmelearn/src/main.jsx:11]: Already using it , somehow missed this one. Ty

File: client/csletmelearn/src/pages/DesignGenerator.jsx (hunk 2 above)
Comment (fdsssawe) [reply client/csletmelearn/src/pages/DesignGenerator.jsx:64]: Probably used it to debug , thank you.

**Pull request merged**